frontend/src/components/Footer.js

```diff
@@ -29,37 +32,146 @@ export default function Footer(){
         //     </a>
         // </div>
         // </div>
-    <div className='footer'>
-        <div className='content'>
-            <div className='socials'>
-                <div className='footer-head'>
+    <Box sx={{aspectRatio: "100 / 12.5",
+              backgroundColor: '#1f1e3a',
+              [theme.breakpoints.down('tablet')]: {
+                aspectRatio: '100/25'
+            }}}>
+
+        <Box sx={{  paddingTop: '30px',
```

Comment:
Ideally you would want to put styles in separate .js file in styles folder and then import that object but its also fine